modules/manage/pages/kubernetes/k-manage-resources.adoc

```diff
@@ -77,34 +107,31 @@ helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --crea
 +
 ```bash
 helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
-  --set resources.cpu.cores=<number-of-cpu-cores>
+  --set resources.cpu.cores=<number-of-cpu-cores> \ <1>
+  --set resources.cpu.overprovisioned=false <2>
 ```
 
 ====
 --
 ======
++
+--
+<1> Set the number of cores to an integer such as `4`. The more cores that are allocated to Redpanda, the more throughput it can support. For a description of CPU resource units, see the https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu[Kubernetes documentation^].
++
+NOTE: If you use fractional values, <<pinning, CPU pinning is disabled>>.
+<2> This default setting allows each Redpanda thread to run on a dedicated core for maximum performance.
+--
 
-IMPORTANT: You cannot decrease the number of CPU cores in a running cluster.
-
-[TIP]
-====
-If you're experimenting with Redpanda in Kubernetes, you can set the number of CPU cores to millicores, which represent fractions of CPU time.
-
-.`cpu-cores.yaml`
-[,yaml]
-----
-resources:
-  cpu:
-    cores: 200m
-----
-====
+. <<qos, Enable the `Guaranteed` QoS class for your Pods that run Redpanda>>.
++
+Only containers that are both part of a `Guaranteed` Pod and have integer CPU requests are assigned exclusive CPUs.
 
+[[pinning]]
 === Disable pinning in shared environments
 
-By default, Redpanda pins its threads to all cores that you allocate to it.
-On a dedicated worker node, CPU pinning allows maximum throughput, but in a development environment, where you may run other Pods on the same worker node as Redpanda, CPU pinning can cause your processes to run slower than usual. In these shared environments, you can tell Redpanda not to pin its threads or memory, and reduce the amount of polling it does to a minimum.
+On a dedicated worker node, CPU pinning allows maximum throughput, but in a development environment, where you may run other Pods on the same worker node as Redpanda, CPU pinning can cause your other processes to run slower than usual. In these shared environments, you can enable overprovisioning so that Redpanda reduces the aggressiveness of CPU usage to prevent resource hogging and ensuring fair CPU time distribution among all processes.
```

Comment:
If I understand correctly, CPU Pinning + CPUSets would mitigate the need for dedicates kubernetes nodes _and_ it would make repanda steal less CPU from other Pods as well? I think it's good to tell users how to disable pinning, if they want, but the premise under which this concept is introduced feels incorrect to me.

Reply:
https://github.com/redpanda-data/docs/pull/434#discussion_r1577728613